steps/tryoutapps-shaman/1.0.0/step.yml

```diff
@@ -0,0 +1,59 @@
+title: TryoutApps Shaman
+summary: |
+  Uploads apps to Infinum's TryoutApps
+description: |
+  Uploads apps to Infinum's TryoutApps using shaman deploy script.
+  Only one build at the time can be uploaded.
+website: https://github.com/jabou/bitrise-step-tryoutapps-shaman
+source_code_url: https://github.com/jabou/bitrise-step-tryoutapps-shaman
+support_url: https://github.com/jabou/bitrise-step-tryoutapps-shaman/issues
```

Comment:
It seems you've moved the step's repository to `https://github.com/infinum/TryoutApps-Shaman-Bitrise-Deploy` but haven't updated the URLs.
Please update them so they point to the new repo.

Reply:
Hi @BirmacherAkos ,

thank you for review! I will fix this URL asap :) 

Should this PR be closed and new one reopened or can I just make fix on our branch and push changes?